src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -71,7 +71,10 @@ message PaymentDetailsRequest {
 // requires X-API-Key header with 'API Key' value
 message RefundRequest {
   string store_id = 1; // UUID of store
-  string order_id = 2;
+  oneof id {
+    string order_id = 2;
+    string link_id = 4; //LinkId created by wombat
```

Comment:
because this property name is going to be exposed to the client sdks, can this be named with the same ID name received from the initial ecom payment creation?